packages/core/src/__integrationtests__/ErrorHandler.spec.ts

```diff
@@ -41,11 +41,12 @@ beforeAll(async () => {
 
 it('records an unknown extrinsic error when transferring less than the existential amount to new identity', async () => {
   await expect(
-    makeTransfer(addressFromRandom(), new BN(1)).then((tx) =>
-      BlockchainUtils.signAndSubmitTx(tx, paymentAccount, {
-        resolveOn: BlockchainUtils.IS_IN_BLOCK,
-        reSign: true,
-      })
+    makeTransfer(addressFromRandom(), new BN(1)).then(
+      (tx: SubmittableExtrinsic) =>
```

Comment:
Why are all these type assignments necessary? Can they not be inferred from the function return type?

Reply:
Apparently not. Lint complains if the type is not specified. Now, this is a test file, so we might ignore this if you deem ok, but otherwise the type annotation seems to be necessary here.

Reply:
Ok I removed the type annotations from the test files.